source/clear-linux/guides/tooling/swupd-guide.rst

```diff
@@ -8,29 +8,29 @@ software.
 
 .. contents::
    :local:
-   :depth: 2
+   :depth: 1
 
 Description
 ***********
 
 :command:`swupd` has two main functions:
 
-#. It manages software replacing APT or YUM, installing bundles
+#. Manages software and replaces APT or YUM, by installing bundles
    rather than packages.
-#. It checks for system updates and installs them.
+#. Checks for system updates and installs them.
 
-:ref:`Bundles <bundles>` are the smallest granularity component that is
-managed by |CL| and contain everything needed to deliver a software
-capability. Rather than downloading a cascade of package dependencies when
-installing a piece of software, a bundle comes with all of its dependencies.
-:command:`swupd` manages overlapping dependencies behind the scenes ensuring
-that all software is compatible across the system.
+:ref:`Bundles <bundles>` contain everything needed to deliver a software
+capability. They are the smallest granularity component that is
+managed by |CL|. A bundle comes with all of its dependencies, rather than
+downloading a cascade of package dependencies when installing a piece of
+software. :command:`swupd` manages overlapping dependencies behind the scenes,
+ensuring that all software is compatible across the system.
 
 Versioning
 ==========
 
-In a traditional distribution, the process of describing current software
-versioning usually involves:
+In a traditional distribution, the software versioning process usually
```

Comment:
The way the new phrasing reads sounds like it is referring to the distribution versioning scheme instead of the versioning of software on the system.